thorlcr/thorutil/thmem.cpp

```diff
@@ -1870,7 +1871,6 @@ class CThorRowCollectorBase : public CSpillable
     }
     ~CThorRowCollectorBase()
     {
-        reset();
```

Comment:
ok, why remove reset() ?

Reply:
reset() was a NOP here. All it used to do was call the spillableRows/spillFiles kill() methods, which naturally happen during destruction anyway.